python/MANIFEST.in

```diff
@@ -0,0 +1,13 @@
+# Cython files
+recursive-include rmm *.pxd
+recursive-include rmm *.pyx
+
+# Build files. Don't use a recursive include on '.' in case the repo is dirty
+include . CMakeLists.txt
+recursive-include rmm CMakeLists.txt
+
+# License
+include LICENSE
+
+# README
+include README
```

Comment:
Think we can drop these

```suggestion
```

Reply:
Ah yes you're right these should be included by default.